ExtractorUtils.Test/unit/Unstable/RuntimeTest.cs

```diff
@@ -97,5 +110,162 @@ public async Task TestRuntimeBuilderStatic()
             var configSource = runtime.GetType().GetField("_configSource", System.Reflection.BindingFlags.NonPublic | System.Reflection.BindingFlags.Instance);
             Assert.IsType<StaticConfigSource<DummyConfig>>(configSource.GetValue(runtime));
         }
+
+        private ConnectionConfig GetConfig()
+        {
+            return new ConnectionConfig
+            {
+                Project = "project",
+                BaseUrl = "https://greenfield.cognitedata.com",
+                Integration = "test-integration",
+                Authentication = new ClientCredentialsConfig
+                {
+                    ClientId = "someId",
+                    ClientSecret = "thisIsASecret",
+                    Scopes = new Cognite.Common.ListOrSpaceSeparated("https://greenfield.cognitedata.com/.default"),
+                    MinTtl = "60s",
+                    Resource = "resource",
+                    Audience = "audience",
+                    TokenUrl = "http://example.url/token",
+                }
+            };
+        }
+
+        [Fact(Timeout = 5000)]
+        public async Task TestBootstrapSinks()
```

Comment:
This seems to test two separate classes sequentially, maybe it should be split into two separate tests?